map.go

```diff
@@ -311,7 +311,7 @@ func (spec *MapSpec) createMap(inner *internal.FD, opts MapOptions, handles *han
 	// In order to support loading these definitions, tolerate the presence of
 	// extra bytes, but require them to be zeroes.
 	if _, err := io.Copy(internal.DiscardZeroes{}, &spec.Extra); err != nil {
-		return nil, errors.New("Extra contains unhandled non-zero bytes, drain before creating map")
+		return nil, errors.New("extra contains unhandled non-zero bytes, drain before creating map")
```

Comment:
This one was capitalized on purpose to point at the `Extra` field. I guess if one happens to encounter this error in the wild, they'd have to go read the lib code anyways to really figure out why.